Orangenspannung/oestersjoen
# O01]add initial countries

# What has changed:
* Add basic data for all europian countries who have different borders (in Europe) 
# How to test:
* Load up the mod and try to play it
* While you should get some warnings, nothing should break
